dockerfile

```
@@ -0,0 +1,22 @@
+ARG PYTHON_VERSION=3.12
+FROM python:${PYTHON_VERSION} AS build
+
+WORKDIR /todoapp
+
+ADD https://github.com/yaroslav-demchenko/devops_todolist.git ./
```

Use build from local directory
```suggestion
COPY . . 
```